realworld-api/src/test/java/com/study/realworld/user/controller/UserControllerTest.java

```diff
@@ -79,7 +85,8 @@ void login() throws Exception {
                    .andExpect(content().contentType(APPLICATION_JSON))
                    .andExpect(jsonPath("$.user.email", is(user.getEmail())))
                    .andExpect(jsonPath("$.user.username", is(user.getUsername())))
-                   .andExpect(jsonPath("$.user.bio", is(user.getBio())));
+                   .andExpect(jsonPath("$.user.bio", is(user.getBio())))
```

Comment:
```suggestion
                   .andExpect(jsonPath("$.user.bio", is(nullValue())))
                   .andExpect(jsonPath("$.user.image", is(nullValue())))
```

```
{
  "user": {
    "email": "jake@jake.jake",
    "token": "jwt.token.here",
    "username": "jake",
    "bio": "I work at statefarm",
    "image": null
  }
}
```
현재 리얼월드 스펙 내에서는 bio와 image는 nullable하게 전달되기 때문에 null값이 들어올 수도 있어요.
test code내에서는 nullable한 값들에 대해서 이렇게 대처할 수 있습니다 :)